docs/howto/run-behind-proxy.rst

```diff
@@ -79,6 +79,12 @@ middleware in your ``airflow.cfg``::
 
     enable_proxy_fix = True
 
+If you need to configure the individual parameters to the ``ProxyFix`` middleware,
+you can set them individually in your ``airflow.cfg``::
+
+    proxy_fix_x_for = 1
```

Comment:
```suggestion
    [webserver]
    proxy_fix_x_for = 1
```

Reply:
I'll fix this for the line above as well, since `enable_proxy_fix` is under `[webserver]` as well